test/acceptance/test/add_tests.go

```diff
@@ -209,24 +194,24 @@ var _ = Describe("Weave GitOps Add Tests", func() {
 		})
 
 		By("And I should see workload for app1 is deployed to the cluster", func() {
-			verifyWorkloadIsDeployed(workloadName1, workloadNamespace1)
+			verifyWorkloadIsDeployed(tip1.workloadName, tip1.workloadNamespace)
 		})
 
 		By("And I should see workload for app2 is deployed to the cluster", func() {
-			verifyWorkloadIsDeployed(workloadName2, workloadNamespace2)
+			verifyWorkloadIsDeployed(tip2.workloadName, tip2.workloadNamespace)
 		})
 
 		By("And repos created have private visibility", func() {
-			Expect(getRepoVisibility(os.Getenv("GITHUB_ORG"), appRepoName1)).Should(ContainSubstring("true"))
-			Expect(getRepoVisibility(os.Getenv("GITHUB_ORG"), appRepoName2)).Should(ContainSubstring("false"))
+			Expect(getRepoVisibility(os.Getenv("GITHUB_ORG"), tip1.appRepoName)).Should(ContainSubstring("true"))
+			Expect(getRepoVisibility(os.Getenv("GITHUB_ORG"), tip2.appRepoName)).Should(ContainSubstring("false"))
```

Comment:
Lots of repeated `os.Getenv` calls. I would read it once from the `env` and store it in a top level variable. That way, changing the variable or logic only needs to be changed once.

Reply:
Fixed.